docs/pages/en-us/Architecture.md

```diff
@@ -13,26 +13,48 @@ banner:
 
 # Architecture
 
-The Audit library logs audit event metadata, belonging to tenant applications, into Elasticsearch.
+The Audit library logs audit event metadata, belonging to tenant applications, into Elasticsearch directly or via the Audit Web Service HTTP REST API.
 
-In order to use Auditing with an application, you must first specify the audit events that the application uses and the parameters that you want to associate with each of the events. The events are specified in an XML file known as the audit event definition file and are used to generate an application-specific, client-side auditing library that sends the application events for auditing to Elasticsearch.
+In order to use Auditing with an application, you must first specify the audit events that the application uses and the parameters that you want to associate with each of the events. The events are specified in an XML file known as the audit event definition file and are used to generate an application-specific, client-side auditing library that sends the application events for auditing to Elasticsearch or the Audit Web Service HTTP REST API.
```

Comment:
Replace 'for auditing to Elasticsearch or the Audit Web Service HTTP REST API' with 'for auditing to either Elasticsearch directly or via the Audit Web Service REST API'.